### PR TITLE
The inevitable genetics gorilla nerf

### PR DESCRIPTION
**Pull request by nianjiilical**
## About The Pull Request

Adds a conditional check to the gorillize() proc for gorillas created via monkey radiation. If the gorilla was an irradiated monkey, its stats are lowered by roughly half.

## Why It's Good For The Game

Gorillas have been incredibly popular the last few days, and it's become apparent that while they do have downsides, they're also relentless killing machines that can break into a lot of areas. In a perfect world they'd be refactored like monkeys were to make them stunnable and whatnot, but haha I can't code.

Rather than completely remove the fun of returning to monke, this PR gives them a hefty stat nerfs to make going ape less "I am a merciless god of validhunting and broken bones" and more "a fun thing to do moreso for the novelty than the powergame". These nerfs ONLY apply to gorillas from irradiated monkeys; admin spawns, wild spawns, gorilla cubes and other methods shouldn't be hit by this.

To be more specific, genetics-created gorillas now have:

-110 HP, down from 220
-5-8 on-hit melee damage, down from 15-18
-10 object damage, down from 20

(i pulled these numbers out of my ass tbh)

## Changelog
:cl:
balance: Gorillas created via unethical amounts of radiation now have noticeably lower combat stats, probably due to all the unethical radiation.
/:cl:

### CODE REVIEW

**Comment by tralezab** [comment]
Something so wrong about gorillas dealing less than 10 damage per punch, here's hoping there's a solution that makes them rarer instead of weaker

**Comment by CASHNOW** [comment]
Did something recently change about how do gorillas? Because I wasn't aware until recently that rads over the required threshold increased the chance to mutate. This made the process pretty instantatious. Before I'd have to keep anti-toxin meds around for the RNG amount of time that it took the monkey to transform into a gorilla. Some times it would only take a minute or two of waiting around. But I had rounds where I spent a very long time waiting to transform, to point of running out of anti-toxin meds and having to figure out a way to communicate with the chemist became a real thing. Nerfing gorilla itself instead of the pathway to achieve it seems really wrong to me, and kinda just relegates it to the funny that *ooga can produce.

File: code/modules/mob/transform_procs.dm [hunk 1]
```diff
@@ -269,12 +269,21 @@
...
 	if(mind)
 		mind.transfer_to(new_gorilla)
 	else
 		new_gorilla.key = key
+
+	if(weaktransform == 1) //if the gorilla was produced via monkey irradiation, nerf it
+		new_gorilla.health = 110
+		new_gorilla.maxHealth = 110
+		new_gorilla.melee_damage_upper = 8
+		new_gorilla.melee_damage_upper = 5
```
Comment (ATH1909) [inline code/modules/mob/transform_procs.dm:284]: you set melee_damage_upper twice here

**Comment by Serxule** [comment]
"me downvote becuz monke good" no. gorillas are way overpowered, "haha near esword level damage baseball bat delimb punch" should never be a thing

**Comment by moocowswag** [comment]
> "me downvote becuz monke good" no. gorillas are way overpowered, "haha near esword level damage baseball bat delimb punch" should never be a thing

"Me grossly exaggerate becuz I ded"  15-18 is slightly more than half esword level damage, delimb tends to only happen to people in crit.


>  relentless killing machines that can break into a lot of areas. 

They cant even break a normal window let alone an airlock without a weapon thats strong enough to actually break it, just like the average crew member.

**Comment by IndieanaJones** [comment]
me downvote because monke good

**Comment by Pugie** [comment]
Yeah this is a half measure, genetics straight up shouldn't have gorillas at all. It's like hulk without having to find the mutations, but also with no damage slowdown.

**Comment by Krysonism** [comment]
> Did something recently change about how to do gorillas? Because I wasn't aware until recently that rads over the required threshold increased the chance to mutate

The gorillize "fix" buffed the chance to become a gorilla massively.

**Comment by Krysonism** [comment]
> 
> 
> Yeah this is a half measure, genetics straight up shouldn't have gorillas at all. It's like hulk without having to find the mutations, but also with no damage slowdown.

Except you cannot wear armour, have 1.5 burn mod, are slower than a human and you won't get revived if you die...

**Comment by D4C-420** [comment]
> Yeah this is a half measure, genetics straight up shouldn't have gorillas at all. It's like hulk without having to find the mutations, but also with no damage slowdown.

Since when did hulks have damage slow down? I would say hulk beats gorilla in usefulness and with proper armor effective health. 

On the changes themselves 110 is technically more than what humans get but given the fact the best healing a gorilla can get is nanites and the diffculty in reviving a dead gorrila outside of a lazurs injector I and the lack of damage reduction i would say the health is lowered a bit too much 150 seems more reasonable. 

On the damage I forget how much a base ball bat is but I think for balance its damage and object damage should be in line with that. The last real issue with gorillas is the delimbing in crit which maked it much harder to fix dead people. I don’t think rad gorillas should have that.

All that given yeah rad rillas should be gimmick.

**Comment by optimumtact** [comment]
#57680

**Comment by Kylerace** [comment]
orange man good!

**Comment by nianjiilical** [comment]
yeah this is a better change thanks orange man

**Comment by maxymax13** [comment]
orange man redemption arc?

**Pull request closed without merging**